translations/Discord-Utilities.phrases.txt

```diff
@@ -419,6 +459,7 @@
 		"lv"		"ILGUMS"
 		"ru"		"ДЛИНА"
 		"ro"		"DURATĂ"
+		"es"		"DURACIÓN"
```

Comment:
Just to be sure, is this "LengthField" meant in a context of duration? or about how long something is?